src/vulkan-renderer/application.cpp

```diff
@@ -188,14 +189,11 @@ void Application::load_shaders() {
 void Application::load_octree_geometry() {
     spdlog::debug("Creating octree geometry.");
 
-    std::shared_ptr<world::Cube> cube = std::make_shared<world::Cube>(2.0f, glm::vec3{0, -1, -1});
-    cube->set_type(world::Cube::Type::OCTANT);
-
-    cube->children()[3]->set_type(world::Cube::Type::EMPTY);
-    cube->children()[5]->set_type(world::Cube::Type::EMPTY);
-    cube->children()[6]->set_type(world::Cube::Type::EMPTY);
-    cube->children()[7]->set_type(world::Cube::Type::EMPTY);
+    // 4: 23 012 | 5: 184352 | 6: 1474162 | 7: 11792978 cubes, DO NOT USE 7!
```

Comment:
We could add a check to the random world generator which just forbids anything greater than 6.

Reply:
If 7 takes too long? Not sure if we really need to prevent the API user to use 7 though.

Reply:
well the method is documented :D if someone wants to try it he can just remove that as well.